open_event/api/sessions.py

```diff
@@ -137,19 +134,33 @@ def fix_payload_post(self, event_id, data):
     def update(self, event_id, service_id, data):
         data = self.validate(data, event_id)
         data_copy = data.copy()
+        st = data_copy.get('session_type')
         data_copy = self.fix_payload_post(event_id, data_copy)
         data = self._delete_fields(data)
         obj = ServiceDAO.update(self, event_id, service_id, data, validate=False)
         obj.track = data_copy['track']
         obj.microlocation = data_copy['microlocation']
         obj.speakers = data_copy['speakers']
-        obj.session_type = data_copy['session_type']
+        obj.session_type = st
```

Comment:
st has a `dict` . I think we will have to create a SessionType object first (using get_or_create). 
Also it will be a good idea to update the sample test data (utils_post_data) since event and sessions POST/PUT API has changed.